cmd/kops/update_cluster.go

```diff
@@ -235,7 +235,15 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 			}
 		} else if firstRun {
 			fmt.Fprintf(sb, "\n")
-			fmt.Fprintf(sb, "Cluster is starting.  It should be ready in a few minutes.\n")
+			fmt.Fprintf(sb, "Cluster is starting.  It should be ready soon!\n")
```

Comment:
Can we stick with "a few minutes"?  People's interpretation of "soon" varies from seconds to hours.